src/main/java/org/snomed/snowstorm/fhir/services/FHIRCodeSystemProvider.java

```diff
@@ -533,7 +533,7 @@ private Parameters validateCode(
 
 				return pMapper.validateCodeResponse(concept, displayValidOrNull, codeSystemVersion);
 			} else {
-				return pMapper.resultFalseWithMessage(code, codeSystemVersion, "The code was not found in the specified code system.");
```

Comment:
Just for my own interest and probable education, why is the message not wanted here?